src/ds/d500/d500-auto-calibration.cpp

```diff
@@ -432,6 +432,12 @@ namespace librealsense
     void d500_auto_calibrated::write_calibration() const
     {
         _calib_engine->write_calibration(_curr_calibration);
+
+        for( auto & cb : _depth_write_callbacks )
+            cb();
+        // Currently only depth calibrations are supported, call correct callbacks based on table type otherwise
+        //for( auto & cb : _color_write_callbacks )
+        //    cb();
```

Comment:
But we do register a callback no?

Reply:
Yes, we register, but currently D500 does not a calibration implemented that changes the table, so it's redundant to call the callback.